sample/build.gradle

```diff
@@ -24,8 +74,8 @@ android {
         applicationId "io.ona.kujaku.sample"
         minSdkVersion 16
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 1
-        versionName "1.0"
+        versionCode getMasterCommitCount()
```

Comment:
How would one test if this works?

Reply:
Merge the code to your local master, make sure to build the app & check your sample module `Android.manifest` file

Add more commits on top of your master and check it again, the version code should change due to the extra commits